package.json

```diff
@@ -13,7 +13,15 @@
     "@oclif/config": "^1.12.10",
     "@oclif/plugin-help": "^2.1.6",
     "@types/bluebird": "^3.5.26",
+    "@types/dotenv": "^6.1.1",
```

Comment:
We certainly already do (since the first commits of this repo). This is probably not obvious from the diff, but visible in the full file of that revision.